### PR TITLE
`config-hyperterm`

### PR DESCRIPTION
**Pull request by hemanth**
## Checklist:

- [x] The title for my package or theme uses its npm title (hyperterm-plugin-example)
- [x] The link for my package or theme uses the npmjs.com link (https://npmjs.com/package/hyperterm-plugin-example)
- [x] There is a visual representation of what my plugin does in the repo. (Plugin: screenshot or gif || Theme: screenshot of the colors within HyperTerm)
- [x] **VERY IMPORTANT:** I've written a short (one sentence) description for my package or theme of why it's awesome and deserves to be on the list.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,6 +67,7 @@ Like `awesome-hyperterm`? Reach out to me and say *hi* on [Twitter](https://twit
 * [hyperterm-electron-highlighter](https://www.npmjs.com/package/hyperterm-electron-highlighter) - Dark - Port of Atom One Dark syntax colors with more vibrant colors.
 * [hyperterm-duotone-darkspace](https://www.npmjs.com/package/hyperterm-duotone-darkspace) - Dark/Orange - A beautiful dark theme inspired by [Duotone Themes](http://simurai.com/projects/2016/01/01/duotone-themes) by [Simurai](http://simurai.com/). More color variations can be found [here](https://www.npmjs.com/search?q=hyperterm-duotone-*).
 * [hyperambient](https://www.npmjs.com/package/hyperambient) – Dark/Light - A theme that reacts to ambient light changes.
+* [config-hyperterm](https://git.io/config-hyperterm) - Easily set/get `hyperterm` config.
```
Comment (matheuss) [inline README.md]: Could you change the link to `https://www.npmjs.com/package/config-hyperterm`? Thanks ☺️

**Comment by hemanth** [comment]
Done.

**Comment by bnb** [comment]
Merged!

**Comment by hemanth** [comment]
Thanks but don't see it in the list yet.

**Comment by matheuss** [comment]
Hm, looks like you misplaced it on the `Themes` list 😅 I'm reverting the merge, so you can correct it, 👌?

**Comment by matheuss** [comment]
Done!

**Comment by hemanth** [comment]
tJeTfLOfl98Mo

**Comment by bnb** [comment]
Thanks @matheuss.

**Pull request merged**